### PR TITLE
Adding port option for Theia template

### PR DESCRIPTION
**Pull request by firatoezcan**
Hey, I tested out the Theia chart from the pre-built templates and my application wants to access port 3000 and that didn't quite work out. Checked quickly and there's a theia cli flag to specify the port so this PR adds that to it.

I also added it to the values-okteto.yaml to make it clear for users why their application may not be running

### CODE REVIEW

File: theia/values-okteto.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # available runtimes: go, python, dart, cpp, java, rust, nodejs
 runtime: nodejs
 
+# This is the port that Theia is running on. Change this if you want to use this port instead
```
Comment (rberrelleza) [inline theia/values-okteto.yaml:4]: Can you put it after `repository`?

**Review by rberrelleza — CHANGES_REQUESTED** [review]
Looks good! 

Could you change the order of the keys in values-okteto.yml? We want to put the more common first.

File: theia/values-okteto.yaml (hunk 1 above)
Comment (firatoezcan) [reply theia/values-okteto.yaml:4]: Sure :)

**Review by pchico83 — APPROVED** [review]

**Review by rberrelleza — APPROVED** [review]

**Comment by rberrelleza** [comment]
Thanks for your contribution @firatoezcan!

**Pull request merged**